README.md

```diff
@@ -1,37 +1,102 @@
-# Storybook Addon Visual Tests
+# Storybook Visual Testing addon
 
-Visual Testing addon with Chromatic
+The Visual testing addon enables you to run visual tests on your stories and compare changes with the latest baselines across multiple viewports and browsers to catch UI regressions early in development without leaving Storybook.
 
-## Installation
 
-Add the `@chromaui/visual-tests` addon, and add it to `main.js`:
+## Prerequisites
 
-```
-yarn add --dev @chromaui/visual-tests
-```
+- Chromatic [account configured](https://www.chromatic.com/docs/setup#sign-up) with access to a project
+- Storybook 7.2 or later
 
+## Getting Started
+
+1. Install this addon by adding the `@chromaui/addon-visual-tests` dependency:
+
+```shell
+yarn add --dev @chromaui/addon-visual-tests
 ```
-// main.js:
-"addons": ["@chromaui/visual-tests"]
-```
 
-### Troubleshooting
+2. Update your Storybook configuration file (e.g., `.storybook/main.js|ts`) to include the addon:
+
+```ts
+// .storybook/main.ts
+
+// Replace your-framework with the framework you are using (e.g., react-webpack5, vue3-vite)
+import type { StorybookConfig } from '@storybook/your-framework';
 
-If you see:
+const config: StorybookConfig = {
+  framework: '@storybook/your-framework',
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  addons: [
+    // Other Storybook addons
+    '@chromaui/addon-visual-tests',
+  ],
+};
 
+export default config;
 ```
-var stringWidth = require('string-width')
-                  ^
 
-Error [ERR_REQUIRE_ESM]: require() of ES Module /Users
+3. Start Storybook and navigate to the Visual Tests panel to run your first visual test with Chromatic!
+
+### Configuration
+
+By default, the addon offers zero-config support to run visual tests with Storybook and Chromatic. However, you can extend your Storybook configuration file (i.e., `.storybook/main.js|ts`) and provide additional options to control how tests are run. Listed below are the available options and examples of how to use them.
+
+
+| Option            | Description                                                                                                                      |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `buildScriptName` | Optional. Defines the custom Storybook build script <br/> `options: { buildScriptName: 'deploy-storybook' }`                     |
+
```

Comment:
I think we should document `projectId`/`projectToken`

Reply:
Good call on this. And yes, we should.

Reply:
Updated with the latest commit